ci-operator/step-registry/baremetalds/packet/teardown/baremetalds-packet-teardown-commands.sh

```diff
@@ -33,7 +33,8 @@ cat > packet-teardown.yaml <<-EOF
       packet_device:
         auth_token: "{{ packet_auth_token }}"
         project_id: "{{ packet_project_id }}"
-        hostnames: "{{ packet_hostname }}" 
+        hostnames: "{{ packet_hostname }}"
+        wait_timeout: 1500 
```

Comment:
From the Ansible module docs it looks like that `wait_timeout` is not considered for a state different from `active` so I guess it's better to remove it

Reply:
Done